charts/rancher-logging/rancher-logging/100.0.1+up3.15.0/templates/loggings/aks/logging.yaml

```diff
@@ -1,104 +1,18 @@
-{{- if .Values.additionalLoggingSources.aks.enabled }}
-apiVersion: logging.banzaicloud.io/v1beta1
-kind: Logging
+{{- define "logging-operator.logging.aks" -}}
+{{- $logPath := "/var/log/azure/kubelet-status.log" -}}
 metadata:
   name: {{ .Release.Name }}-aks
-  namespace: {{ .Release.Namespace }}
-  labels:
-{{ include "logging-operator.labels" . | indent 4 }}
 spec:
-  controlNamespace: {{ .Release.Namespace }}
   fluentbit:
-    image:
-      repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}
-      tag: {{ .Values.images.fluentbit.tag }}
+    disableKubernetesFilter: true
+    extraVolumeMounts:
+      - source: {{ $logPath }}
+        destination: {{ $logPath }}
+        readOnly: true
```

Comment:
This looks new vs. the generic template. What are these used for?

Reply:
we were not properly mounting files before so these loggings were never working.